.pre-commit-config.yaml

```diff
@@ -1,7 +1,11 @@
 ---
+default_language_version:
```

Comment:
Do we want this backported to the skeletons?

Reply:
I think we do.

I was actually getting into some problems with parse issues with the linters.  Despite being in a python 3 virtual environment some of the linters were pulling in python 2.    

Specifically it was upset about this: https://github.com/cisagov/gophish-docker/blob/4f641a078aa26ab82e8f33efdc513a6eb2a2eeac/tests/container_test.py#L29-L32